src/components/AuctionStatus/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+import { TokenCode, Balance } from 'types'
+
+import claim from 'assets/claim.svg'
+
+// TODO: move to global types
+export enum Status {
+  INIT = 'initialising',
+  PLANNED = 'planned',
+  ACTIVE = 'active',
+  ENDED = 'ended',
+}
+
+export interface AuctionStatusProps {
+  sellToken: TokenCode,
+  buyToken: TokenCode,
+  buyAmount: Balance,
+  timeLeft: string,
+  status: Status
+}
+
+const capitalize = (str: string) => str && (str[0].toUpperCase() + str.slice(1))
+
+const AuctionStatus: React.SFC<AuctionStatusProps> = ({ sellToken, buyToken, buyAmount, timeLeft, status }) => (
+  <div className="auctionStatus">
+    <span>
+      <small>AUCTION</small>
+      <big>{sellToken}/{buyToken}</big>
+    </span>
+
+    <span>
+      <small>STATUS</small>
+      <big data-status={status}>{capitalize(status)}</big>
+    </span>
+
+    <span>
+      {
+        status === Status.ACTIVE ? [
```

Comment:
May be fine here as the conditional choices are rather limited but I prefer to create ENUM components and split up the components
ex. ```const AuctionStatus = ({status}) => {
  [status.ACTIVE]: <AuctionActive>,
  [status.XYZ]: <AuctionXYZ>
}```